public/assets/.sprockets-manifest-8555c949cd6592490c70cbd0375ea8d7.json

```diff
@@ -0,0 +1 @@
+{"files":{"application-28fa8e3f5a442dcf5ed50188edad6735f09b92339a7c7e77a8c089b5f0f87703.js":{"logical_path":"application.js","mtime":"2018-03-10T12:22:33+01:00","size":446895,"digest":"28fa8e3f5a442dcf5ed50188edad6735f09b92339a7c7e77a8c089b5f0f87703","integrity":"sha256-KPqOP1pELc9e1QGI7a1nNfCbkjOafH53qMCJtfD4dwM="},"application-c3edde78272ef25d9c3ea3af466e8e0b8f48861e44dedd2a11703f9942d16614.css":{"logical_path":"application.css","mtime":"2018-03-10T12:21:55+01:00","size":121152,"digest":"c3edde78272ef25d9c3ea3af466e8e0b8f48861e44dedd2a11703f9942d16614","integrity":"sha256-w+3eeCcu8l2cPqOvRm6OC49Ihh5E3t0qEXA/mULRZhQ="},"glyphicons-halflings-regular-13634da87d9e23f8c3ed9108ce1724d183a39ad072e73e1b3d8cbf646d2d0407.eot":{"logical_path":"glyphicons-halflings-regular.eot","mtime":"2018-02-22T16:24:05+01:00","size":20127,"digest":"13634da87d9e23f8c3ed9108ce1724d183a39ad072e73e1b3d8cbf646d2d0407","integrity":"sha256-E2NNqH2eI/jD7ZEIzhck0YOjmtBy5z4bPYy/ZG0tBAc="},"glyphicons-halflings-regular-a26394f7ede100ca118eff2eda08596275a9839b959c226e15439557a5a80742.woff":{"logical_path":"glyphicons-halflings-regular.woff","mtime":"2018-02-22T16:24:05+01:00","size":23424,"digest":"a26394f7ede100ca118eff2eda08596275a9839b959c226e15439557a5a80742","integrity":"sha256-omOU9+3hAMoRjv8u2ghZYnWpg5uVnCJuFUOVV6WoB0I="},"glyphicons-halflings-regular-e395044093757d82afcb138957d06a1ea9361bdcf0b442d06a18a8051af57456.ttf":{"logical_path":"glyphicons-halflings-regular.ttf","mtime":"2018-02-22T16:24:05+01:00","size":45404,"digest":"e395044093757d82afcb138957d06a1ea9361bdcf0b442d06a18a8051af57456","integrity":"sha256-45UEQJN1fYKvyxOJV9BqHqk2G9zwtELQahioBRr1dFY="},"glyphicons-halflings-regular-42f60659d265c1a3c30f9fa42abcbb56bd4a53af4d83d316d6dd7a36903c43e5.svg":{"logical_path":"glyphicons-halflings-regular.svg","mtime":"2018-02-22T16:24:05+01:00","size":108738,"digest":"42f60659d265c1a3c30f9fa42abcbb56bd4a53af4d83d316d6dd7a36903c43e5","integrity":"sha256-QvYGWdJlwaPDD5+kKry7Vr1KU69Ng9MW1t16NpA8Q+U="}},"assets":{"application.js":"application-28fa8e3f5a442dcf5ed50188edad6735f09b92339a7c7e77a8c089b5f0f87703.js","application.css":"application-c3edde78272ef25d9c3ea3af466e8e0b8f48861e44dedd2a11703f9942d16614.css","glyphicons-halflings-regular.eot":"glyphicons-halflings-regular-13634da87d9e23f8c3ed9108ce1724d183a39ad072e73e1b3d8cbf646d2d0407.eot","glyphicons-halflings-regular.woff":"glyphicons-halflings-regular-a26394f7ede100ca118eff2eda08596275a9839b959c226e15439557a5a80742.woff","glyphicons-halflings-regular.ttf":"glyphicons-halflings-regular-e395044093757d82afcb138957d06a1ea9361bdcf0b442d06a18a8051af57456.ttf","glyphicons-halflings-regular.svg":"glyphicons-halflings-regular-42f60659d265c1a3c30f9fa42abcbb56bd4a53af4d83d316d6dd7a36903c43e5.svg"}}
```

Comment:
I don't think these assets files should be included in this commit. You can leave it here for now, but it's priceless to keep your commits atomic.